lib/ufe/ProxyShapeHierarchyHandler.cpp

```diff
@@ -17,15 +17,15 @@
 #include "ProxyShapeHierarchyHandler.h"
 #include "Utils.h"
 
+#include <ufe/ProxyShapeHierarchy.h>
+
 MAYAUSD_NS_DEF {
 namespace ufe {
 
 ProxyShapeHierarchyHandler::ProxyShapeHierarchyHandler(Ufe::HierarchyHandler::Ptr mayaHierarchyHandler)
 	: Ufe::HierarchyHandler()
 	, fMayaHierarchyHandler(mayaHierarchyHandler)
-{
-	fProxyShapeHierarchy = ProxyShapeHierarchy::create(mayaHierarchyHandler);
```

Comment:
ProxyShapeHierarchy stores a UFE scene item, so best create it as needed, otherwise we'd need observation to keep the scene item's path up to date, which gratuitous complexity.